Library/Formula/logstash.rb

```diff
@@ -20,6 +22,6 @@ def caveats; <<-EOS.undent
 
   test do
     system "#{bin}/logstash", "--version"
-    system "/bin/echo 'hello world' | #{bin}/logstash agent -e 'input { stdin { type => stdin } } output { stdout { codec => rubydebug } }'"
+    system "/bin/echo", "'hello world' | #{bin}/logstash agent -e 'input { stdin { type => stdin } } output { stdout { codec => rubydebug } }'"
```

Comment:
It'd be nice to kick it onto two lines instead of the one. It's currently making the formula rather wide.